packages/common/src/store/account/selectors.ts

```diff
@@ -177,11 +177,13 @@ export const getAccountWithAlbums = createSelector(
   }
 )
 
-export const getAccountAlbums = createSelector(
```

Comment:
is this selector used anywhere else?

Reply:
actually it is, ty for catch

Reply:
thought i added it but no

Reply:
omg actually no it's not! i did add it! there was another selector in `saved` stuff that had same name